### PR TITLE
Added new features to AXtension Content Gate.

### PR DESCRIPTION
**Pull request by JoepKillaars**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

Please note that we are experiencing a significant delay due to COVID delays, unanticipated team leaves, and high volume of connector submissions. 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by JoepKillaars** [comment]
@natalie-pienkowska Can you help us out with the apiproperties.json errors ? 

This file was approved in previous versions, but as soon as there is only a single newline change to this file, the validations are run and they fail. I cannot figure out what is wrong here.

**Comment by Amjed-Ayoub** [comment]
@JoepKillaars thank you for reaching out. Our Certification Engineer @sriyen will review and five you feedback soon.
Thank you for your patience.

File: certified-connectors/AXtension Content Gate/apiDefinition.swagger.json [hunk 1]
```diff
@@ -2442,6 +2522,135 @@
...
+                    "type": "string",
+                    "description": "apiName"
+                  },
+                  "propertyType": {
+                    "type": "string",
+                    "description": "propertyType"
+                  },
+                  "allowEditOnCreate": {
+                    "type": "boolean",
+                    "description": "allowEditOnCreate"
```
Comment (sriyen-msft) [inline certified-connectors/AXtension Content Gate/apiDefinition.swagger.json:2559]: @JoepKillaars All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by JoepKillaars** [comment]
@Amjed-Ayoub Thanks for assisting us.

@sriyen-msft Thanks for your feedback regarding the swagger file. I will look at those descriptions, although they are for internal methods. 

But my original question was not about the swagger file, it was about the apiproperties file, which fails the build check and I have no clue as why that is. Hope we can resolve this soon.

File: certified-connectors/AXtension Content Gate/apiDefinition.swagger.json (hunk 1 above)
Comment (JoepKillaars) [reply certified-connectors/AXtension Content Gate/apiDefinition.swagger.json]: Adjusted swagger file.

Comment (sriyen-msft) [reply certified-connectors/AXtension Content Gate/apiDefinition.swagger.json]: @JoepKillaars There are few other descriptions having grammatical issues. Please correct all of them in the swagger.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@JoepKillaars There are few other descriptions having grammatical issues. Please correct all of them in the swagger.

**Comment by JoepKillaars** [comment]
@sriyen-msft I will fix the other descriptions in the swagger file. But the original question remains about the apiproperties.json file, can you please assist me in correcting that file ?

**Comment by JoepKillaars** [comment]
@Amjed-Ayoub @sriyen-msft Any update on this?

**Review by v-eisljami — APPROVED** [review]

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**